pkgs/by-name/ta/tabby-terminal/package.nix

```diff
@@ -0,0 +1,242 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  git,
+  patch-package,
+  electron_29,
+  prefetch-yarn-deps,
+  fixup-yarn-lock,
+  python3,
+  makeWrapper,
+  http-server,
+  fontconfig,
+  pkg-config,
+  libsecret,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+
+let
+  tabby = rec {
+    pname = "tabby-terminal";
+    version = "1.0.215";
+
+    src = fetchFromGitHub {
+      owner = "Eugeny";
+      repo = "tabby";
+      tag = "v${version}";
+      hash = "sha256-l3Hyt3e1edJjb7vL66JDllkpHwTU5efsNbgJ5Pcvn60=";
+      leaveDotGit = true;
+      deepClone = true;
+    };
+
+    meta = {
+      description = "A terminal for a more modern age";
+      homepage = "https://tabby.sh";
+      license = lib.licenses.mit;
+      mainProgram = "tabby";
+      maintainers = with lib.maintainers; [ geodic ];
+      platforms = [
+        "x86_64-linux"
+        "aarch64-linux"
+      ];
+    };
+
+    # We need to fix the argv logic as tabby only handles it for the node executable, but we are running it with electron
+    patches = [
+      ./fix-argv-prefix-splice.patch
+      ./fix-app-version.patch
+    ];
+
+    desktopItems = [
+      (makeDesktopItem {
+        name = "tabby";
+        desktopName = "Tabby";
+        exec = "tabby %u";
+        comment = "A terminal for a more modern age";
+        icon = "tabby";
+        categories = [
+          "Utility"
+          "TerminalEmulator"
+          "System"
+        ];
+      })
+    ];
+
+    electronVersion = "29.4.6";
```

Comment:
Have you tried using the latest Electron version? If it works then there's no reason to do this

Reply:
Using latest electron requires node-abi to be updated... Would a patch be preferrable or the jq approach as used in the other package.json modification?